azext_iot/product/_help.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+Help definitions for Product Certification commands.
+"""
+
+from knack.help_files import helps
+
+
+def load_help():
+    helps[
+        "iot product"
+    ] = """
+        type: group
+        short-summary: Manage device testing for product certification
+    """
+    # certification requirements
+    helps[
+        "iot product requirement"
+    ] = """
+        type: group
+        short-summary: Manage product certification requirements
+    """
+    helps[
+        "iot product requirement list"
+    ] = """
+        type: command
+        short-summary: Get certification requirements
+        long-summary: |
+                      Discover information about provisioning attestation methods that are supported for each badge type
```

Comment:
Just a note (not necessarily suggesting a change) currently long summary does not get rendered on the MS docs reference page - only short summary.